### DISCUSSION TITLE
Java (i guess?) error and crash ETIMEDOUT

### DISCUSSION THREAD

**Discussion by Boccas95** [SnosMe/awakened-poe-trade#1267] [Q&A]
Hello there, i have this problem since 2 days ago update.
It seems to happen if i press ALT too much. Can someone help me? Need to restart awakened to make it work again

This is the error i get
![Senza titolo](https://github.com/SnosMe/awakened-poe-trade/assets/126776400/fefa4586-70f6-4c58-9d40-a12eee4543d8)

**Comment by SnosMe** [comment]
App tries to connect to that IPv6 address for poe.ninja I assume, but it's blocked. Idk who blocks, but you can try VPN

**Reply by Boccas95** [reply]
Is there a way to understand what causes it? it s pretty a problem and i cant use vpn, it really happens every 3 min

**Comment by hongchenduzhe** [comment]
A more important question is, maybe we should do an exception handle so that the program doesn't pop up this pop-up?

**Reply by SnosMe** [reply]
Done
